src/parse-options.js

```diff
@@ -1,18 +1,33 @@
 import {resolve} from 'path'
+import styles from 'chalk'
 
 const DEFAULT_OPTIONS = {
   nodeBin: 'node',
   maxAttempts: 3,
   protractorArgs: [],
+  // set color to true to use a default color or set to one of the colors available at 'chalk' - https://github.com/chalk/ansi-styles#colors
+  color: null,
```

Comment:
We will need a way to disable coloring altogether. So we still have to check for `false`.